app/Filament/Admin/Resources/ResultResource.php

```diff
@@ -52,10 +52,10 @@ public static function table(Table $table): Table
                 TextColumn::make('name')
                     ->label(__('nomenclature.labels.result_name')),
 
-                TextColumn::make('institution_count')
+                TextColumn::make('institutions_count')
                     ->label(__('nomenclature.labels.institutions')),
 
-                TextColumn::make('center_count')
+                TextColumn::make('organizations_count')
                     ->label(__('nomenclature.labels.centers')),
```

Comment:
Relation cu methoda ->count()

Reply:
relatia intre Result si Organization e prin InterventionPlanResult -> InterventionPlan -> Beneficiary -> Organization. la fel e si in cazult Benefit